e2etest/newe2e_task_runazcopy.go

```diff
@@ -318,7 +319,7 @@ func RunAzCopy(a ScenarioAsserter, commandSpec AzCopyCommand) (AzCopyStdout, *Az
 			}
 		case commandSpec.Verb == AzCopyVerbList:
 			out = &AzCopyParsedListStdout{}
-		case commandSpec.Verb == AzCopyVerbJobsList:
+		case commandSpec.Verb == AzCopyVerbJobs && len(commandSpec.PositionalArgs) != 0 && commandSpec.PositionalArgs[0] == "list":
```

Comment:
hmm, I know I mentioned I dont like the duplicate AzCopyVerb for jobs, but what do you think about just making the AzCopyVerb "jobs list" and "jobs clean"? Then leave positional args empty, and we can have separate verbs? We can leave the positional args for actual src/dest type args instead of azcopy command strings.

Reply:
Got it, ok this seems fine to me then.